data/get_multipole_data.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import os
+import shutil
+import subprocess
+import sys
+import tarfile
+import glob
+import hashlib
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-b', '--batch', action = 'store_true',
+                    help = 'supresses standard in')
```

Comment:
No spaces around the = sign for action and help (you can fix in get_nndc_data.py too if you wish)